### DISCUSSION TITLE
Getting Bad State - no test targets to run

### DISCUSSION THREAD

**Discussion by akagupta9** [leancodepl/patrol#795] [Q&A] [answered]
Bad state: no test targets to run
#0      FlutterTestRunner.run (package:patrol_cli/src/features/drive/flutter_test_runner.dart:105:7)
#1      DriveCommand.execute (package:patrol_cli/src/features/drive/drive_command.dart:310:39)
<asynchronous suspension>
#2      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
<asynchronous suspension>
#3      PatrolCommandRunner.runCommand (package:patrol_cli/src/command_runner.dart:291:18)
<asynchronous suspension>
#4      PatrolCommandRunner.run (package:patrol_cli/src/command_runner.dart:235:18)
<asynchronous suspension>
#5      patrolCommandRunner (package:patrol_cli/src/command_runner.dart:52:14)
<asynchronous suspension>

**Comment by bartekpacia** [comment, marked as answer]
Hi!

Can you show me contents of your `integration_test` directory?

**Reply by akagupta9** [reply]
<img width="1440" alt="Screenshot 2023-01-19 at 6 53 29 PM" src="https://user-images.githubusercontent.com/43808980/213453966-8f57fcc7-d9f2-4a4f-a582-1ce4f2969348.png">

**Reply by akagupta9** [reply]
Running using 

patrol drive --flavor "$FLAVOR" --dart-define=ENVIRONMENT="$ENV" --dart-define=SQUAD="CUSTOM" --dart-define=CUSTOM_URL_CONFIG="$2" integration_test/patrol_integration_tests/tests/monetization.upgrade.tests.dart -d "$DEVICE"

**Reply by bartekpacia** [reply]
Your Dart file has to end with `_test.dart`.

**Reply by akagupta9** [reply]
Thanks, It worked.
